bundle/src/main/java/com/adobe/acs/commons/users/impl/EnsureAuthorizableManager.java

```diff
@@ -5,12 +5,12 @@
 import com.adobe.granite.jmx.annotation.Name;
 
 @ProviderType
-@Description("ACS AEM Commons - Ensure Service User MBean")
-public interface EnsureServiceUserManager {
+@Description("ACS AEM Commons - Ensure Authorizable MBean")
+public interface EnsureAuthorizableManager {
 
-    @Description("Execute all Ensure Service User configurations")
+    @Description("Execute all Ensure Service User & Ensure Group configurations")
```

Comment:
Curious as why one description uses `&` and the other 'and' :) .. my OCD coming out perhaps.